styles.css

```diff
@@ -0,0 +1,540 @@
+}
+
+.grid ul{
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+  
+.About{
+    background-color: #FFE4E1;
+    text-align: center;
+    padding: 4rem 25rem; 
+}
+
+h3 span{
+    color: black;
+    font-size: 24px; 
+    text-align: center;
+    font-weight: 700;
+    line-height: 1.34rem;
+}
+
+.grid{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+    justify-content: center;
+    width: min(640px, 85vw);
+    margin-inline: auto;
+}
+
+.circle-1{
+    height: 100px; 
+    width: 100px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-top: -300px; 
+    margin-left: 200px; 
+}
+
+.circle-2{
+    height: 200px; 
+    width: 200px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-left: 950px; 
+    margin-top: -400px; 
+}
+
+.circle-3{
+    height: 300px; 
+    width: 300px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-top: -100px; 
+    margin-left: -500px; 
+}
+
+.circle-4{
+    height: 100px; 
+    width: 100px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-top: -450px; 
+    margin-left: -150px; 
+    z-index: 10;  
+}
+
+.circle-5{
+    height: 250px; 
+    width: 250px; 
+    background-color:#FFB6C1; 
+    border-radius: 5000px; 
+    position: absolute; 
+    opacity: 0.3; 
+    margin-left: 500px; 
+    
+
+}
+
+.Skills{
+    background-color: #1c232e;
+    margin: 0; 
+    padding: 4rem 2rem; 
+    text-align: center;
+    font-size: 70px; 
+    position: absolute;
+    z-index: 50;
+}
+
+.icon-text{
+    font-size: 20px; 
+    font-family: Montserrat, sans-serif;
+    font-weight: 550;
+    text-align: center; 
+}
+
+.Skills i{
+    min-width: 200px; 
+}
+
+.gallery{
+    width: 900px; 
+    display: flex; 
+    overflow-x: scroll; 
+}
+
+.gallery div{
+    width: 100%; 
+    display: flex; 
+    padding: 10px;
+    flex: none;  
+}
+
+.gallery div i{
+    width: 100%; 
+    margin: 10px;
+}
+
+.gallery::-webkit-scrollbar{
+    display: none; 
+}
+
+.gallery-wrap{
+    display: flex; 
+    align-items: center; 
+    justify-content: center; 
+    margin: 0% auto; 
+}
+
+#backBtn, #nextBtn{
+    width: 50px; 
+    cursor: pointer; 
+    margin: 40px; 
+    color:#373d47; 
+    font-size: 50px;
+    
+}
+
+#backBtn{
+    margin-left: 20px; 
+}
+
+#nextBtn{
+    margin-left: 20px; 
+}
+
+#nextBtn:hover{
+    color:#FFD700; 
+}
+
+#backBtn:hover{
+    color:#FFD700; 
+}
+
+.projects{
+    margin: 0; 
+    padding: 8rem 2rem; 
+    text-align: center;
+    background-color: white;
+}
+
+.projects > h5{
+    color: black; 
+    position: relative;
+    font-size: 24px; 
+    text-align: center;
+    font-weight: 700;
+    line-height: 1.34rem;
+    margin-top: 300px;
+}
+
+@media (max-width: 800px){
```

Comment:
The entire website needs to be responsive. Currently only looks good full screen. Need help with using @media{}